input/data/2012/2012ApJ...754L..10A/tev-000030-lc.ecsv

```diff
@@ -0,0 +1,26 @@
+# %ECSV 0.9
+# ---
+# datatype:
+# - {name: e_min, unit: TeV, datatype: float64}
+# - {name: time, unit: MJD, datatype: float64}
+# - {name: flux, unit: ph / (cm2 s), datatype: float64}
+# - {name: flux_err, unit: ph / (cm2 s), datatype: float64}
+# meta: !!omap
+# - data_type: lc
+# - source_id: 30
+# - source_name: HESS J0632+057
+# - paper_id: 2012ApJ...754L..10A
+# - {SED_TYPE: flux}
+# - UL_CONF: 0.99
+# - comments: |
+# MAGIC light curve
```

Comment:
Some of your ECSV files can't be read by Astropy table:
https://travis-ci.org/gammapy/gamma-cat/builds/187298896

To reproduce, you can run `./make.py check` locally, or directly try to read your ECSV file with Astropy Table:
```python
from astropy.table import Table
Table.read('https://raw.githubusercontent.com/GernotMaier/gamma-cat/3ea707cf54e129a0ee5d057db5939e663446d34d/input/data/2012/2012ApJ...754L..10A/tev-000030-lc.ecsv', format='ascii.ecsv')
# Error
```
I didn't look in detail, but I think it might just be that you need to indent the comments field properly. This should be OK:
```
# - comments: |
#     MAGIC light curve
```